src/api/forms/form-metadata-repository.js

```diff
@@ -63,6 +63,21 @@ export async function create(document) {
   }
 }
 
+/**
+ * Update a document in the database
+ * @param {string} formId - ID of the form
+ * @param {import('mongodb').UpdateFilter<FormMetadataDocument>} update - form metadata document update filter
```

Comment:
Mind if we keep these imports at the bottom of the file?

Mainly in preparation for `@import` coming to TypeScript 5.5

```suggestion
 * @param {UpdateFilter<FormMetadataDocument>} update - form metadata document update filter
```

<br>

For now we'll need to declare the generic ourselves, something like:

```js
/**
 * @template {object} Schema
 * @typedef {import('mongodb').UpdateFilter<Schema>} UpdateFilter
 */
```

Reply:
Resolved in 143e9cd20fb194525b54826b417f88b54311d695